src/System.Private.CoreLib/shared/System.Private.CoreLib.Shared.projitems

```diff
@@ -893,4 +893,18 @@
     <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\Intrinsics\X86\Sse42.PlatformNotSupported.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\Intrinsics\X86\Ssse3.PlatformNotSupported.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(FeatureHardwareIntrinsics)' == 'true' AND '$(Platform)' == 'arm64'">
+    <Compile Include="$(BclSourcesRoot)\System\Runtime\Intrinsics\Arm\Arm64\Aes.cs" />
```

Comment:
These should all be `$(MSBuildThisFileDirectory)` instead of `$(BclSourcesRoot)`, right?  Like the ones above.